frontend/src/lib-customizations/espoo/employee/assets/i18n/fi.ts

```diff
@@ -2653,6 +2653,7 @@ export const fi = {
       daycareType: 'Varhaiskasvatuksen tyyppi puuttuu',
       capacity: 'Kapasiteetti on virheellinen',
       costCenter: 'Kustannuspaikka puuttuu',
+      url: 'URL-osoitteessa pitää olla http(s)://-etuliite',
```

Comment:
As people editing these might not be familiar with what this is, should we be even more explicit like:
```suggestion
      url: 'URL-osoitteessa pitää olla https://- tai http://-etuliite',
```

Reply:
Yeah, good point.